spec/mail/fields/unstructured_field_spec.rb

```diff
@@ -150,7 +151,7 @@
 
     it "should fold itself if it is non us-ascii" do
       @original = $KCODE if RUBY_VERSION < '1.9'
-      string = "This is あ really long string This is あ really long string This is あ really long string This is あ really long string This is あ really long string"
+      string = String.new("This is あ really long string This is あ really long string This is あ really long string This is あ really long string This is あ really long string")
       @field = Mail::UnstructuredField.new("Subject", string)
       if string.respond_to?(:force_encoding)
         string = string.force_encoding('UTF-8')
```

Comment:
calling #dup moves it off to the right end of a really long string and immediately clear that it's not a frozen literal - can change if really wanted